src/path.cc

```diff
@@ -269,6 +269,43 @@ std::string PathResolve(Environment* env,
 void ToNamespacedPath(Environment* env, BufferValue* path) {
 #ifdef _WIN32
   if (path->length() == 0) return;
+
+  const std::vector<std::regex> windowDevicePatterns = {
```

Comment:
I suggest a persistent vector to avoid recreating it on each call and improve performance.
```suggestion
  static const std::vector<std::regex> windowsDevicePatterns = {
```

Reply:
Thank you for your through suggestions. I made changes according to your suggestions.